reference/constraints.rst

```diff
@@ -50,6 +50,7 @@ Validation Constraints Reference
    constraints/Currency
    constraints/Luhn
    constraints/Iban
+   constraints/Bic
```

Comment:
But let's do that in another PR, which is based on 2.3. For now, only update the newly added code examples